### PR TITLE
IK-620 Embed video docs

### PR DESCRIPTION
**Pull request by ankur-dwivedi**

### CODE REVIEW

File: features/embed-video.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 5.43.10 AM.png" alt=""><figcaption><p>right-click on video file</p></figcaption></figure>
+
+</div>
+
+#### File detail page
+
+Open the file detail page for any published video file and select the embed option from the header section.
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 5.52.53 AM.png" alt=""><figcaption><p>FIle detail page</p></figcaption></figure>
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 5.52.53 AM.png" alt=""><figcaption><p>File detail page</p></figcaption></figure>
```

File: features/embed-video.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 6.29.16 AM.png" alt=""><figcaption></figcaption></figure>
+
+The custom option is also available which lets a user enter a transformation string, which could be used to add multiple [supported video transformation parameters](image-transformations/resize-crop-and-other-transformations.md) that are also supported by [Adaptive bitrate streaming-enabled](video-transformation/adaptive-bitrate-streaming.md) video.
+
+{% hint style="info" %}
+The transformation string must contain `sr-<representations> transformation.`
+{% endhint %}
+
+**Named transformation** could also be added in a custom transformation string as `tr:n-named_transform`\
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
**Named transformations** can be incorporated as part of custom transformation by specifying `tr:n-named_transform`
```

File: features/embed-video.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+#### Enable ABS
+
+Select the enable ABS option to activate  [Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) for embedding video then select the protocol and resolution option from the dropdown.
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 6.29.16 AM.png" alt=""><figcaption></figcaption></figure>
+
+The custom option is also available which lets a user enter a transformation string, which could be used to add multiple [supported video transformation parameters](image-transformations/resize-crop-and-other-transformations.md) that are also supported by [Adaptive bitrate streaming-enabled](video-transformation/adaptive-bitrate-streaming.md) video.
+
+{% hint style="info" %}
+The transformation string must contain `sr-<representations> transformation.`
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
The transformation string must contain `sr-<representations>` transformation.
```

File: features/embed-video.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+The expiration time option is only available for private video files.
+{% endhint %}
+
+An embedded video is by default set to never expire unselect the check box to select an expiration date and time.
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 6.19.36 AM.png" alt=""><figcaption><p>Expiration time</p></figcaption></figure>
+
+#### Enable ABS
+
+Select the enable ABS option to activate  [Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) for embedding video then select the protocol and resolution option from the dropdown.
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
Select the 'Enable ABS' option to activate [Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) for embedding video then select the protocol and resolution option from the dropdown.
```

File: features/embed-video.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+The following embed options are available&#x20;
+
+#### Expiration time
+
+{% hint style="info" %}
+The expiration time option is only available for private video files.
+{% endhint %}
+
+An embedded video is by default set to never expire unselect the check box to select an expiration date and time.
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
By default, an embedded video is set to never expire. Unselect the checkbox to set an expiration date and time.
```

File: features/embed-video.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 5.52.53 AM.png" alt=""><figcaption><p>FIle detail page</p></figcaption></figure>
+
+### Configure embed video code
+
+On clicking on the embed option it will open the embed video model.
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 6.07.37 AM.png" alt=""><figcaption><p>Embed video modal</p></figcaption></figure>
+
+The following embed options are available&#x20;
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
The following embed options are available:
```

File: features/embed-video.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+#### File detail page
+
+Open the file detail page for any published video file and select the embed option from the header section.
+
+<figure><img src="../.gitbook/assets/Screenshot 2023-10-09 at 5.52.53 AM.png" alt=""><figcaption><p>FIle detail page</p></figcaption></figure>
+
+### Configure embed video code
+
+On clicking on the embed option it will open the embed video model.
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
Clicking the embed option will open the embed video modal.
```

File: features/embed-video.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+Embed video code could be generated for any video from the media library dashboard and from the files detail page.
+
+{% hint style="info" %}
+The embed feature is only available for published video files
+{% endhint %}
+
+#### Media library dashboard
+
+To generate embed video code from the media library dashboard hover on the lower right section of the video or right-click on a video file and select the embed option.\
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
To generate the embed video code from the media library dashboard, hover over the lower right section of the video or right-click on the video file and select the 'embed' option.
```

File: features/embed-video.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+{% endcode %}
+
+[Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) can be enabled  for embedding video  for both HTTP Live Streaming (HLS) and Dynamic Adaptive Streaming over HTTP (MPEG-DASH) protocol
+
+### How do you generate embed video code?
+
+Embed video code could be generated for any video from the media library dashboard and from the files detail page.
+
+{% hint style="info" %}
+The embed feature is only available for published video files
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
The embed feature is only available for published video files.
```

File: features/embed-video.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+```
+<iframe width="560" height="315" src="https://stage.imagekit.io/player/embed/se1fciwn2/test_1.5_Y9p3EI9nq.mp4?updatedAt=1693430496863&ik-s=7e68dc5e139ba18ce8de73da30d7fadc736efbae&thumbnail=https%3A%2F%2Fstage-ik.imagekit.io%2Fse1fciwn2%2Ftest_1.5_Y9p3EI9nq.mp4%2Fik-thumbnail.jpg" title="ImageKit video player" frameBorder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share; fullscreen"> </iframe>
+```
+{% endcode %}
+
+[Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) can be enabled  for embedding video  for both HTTP Live Streaming (HLS) and Dynamic Adaptive Streaming over HTTP (MPEG-DASH) protocol
+
+### How do you generate embed video code?
+
+Embed video code could be generated for any video from the media library dashboard and from the files detail page.
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
Embed video code can be generated for any video from the media library dashboard or from the file details page.
```

File: features/embed-video.md [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+
+An embedded video lets you use the video directly on your platform from Imagkit, without any additional video player. An embed video code would  usually look like this
+
+{% code overflow="wrap" %}
+```
+<iframe width="560" height="315" src="https://stage.imagekit.io/player/embed/se1fciwn2/test_1.5_Y9p3EI9nq.mp4?updatedAt=1693430496863&ik-s=7e68dc5e139ba18ce8de73da30d7fadc736efbae&thumbnail=https%3A%2F%2Fstage-ik.imagekit.io%2Fse1fciwn2%2Ftest_1.5_Y9p3EI9nq.mp4%2Fik-thumbnail.jpg" title="ImageKit video player" frameBorder="0" allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share; fullscreen"> </iframe>
+```
+{% endcode %}
+
+[Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) can be enabled  for embedding video  for both HTTP Live Streaming (HLS) and Dynamic Adaptive Streaming over HTTP (MPEG-DASH) protocol
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
[Adaptive bitrate streaming](video-transformation/adaptive-bitrate-streaming.md) (ABS) can be enabled  for embedding video using both HTTP Live Streaming (HLS) and Dynamic Adaptive Streaming over HTTP (MPEG-DASH) protocol.
```

**Review by ahnv — CHANGES_REQUESTED** [review]

File: features/embed-video.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
+# Embed video
+
+An embedded video lets you use the video directly on your platform from Imagkit, without any additional video player. An embed video code would  usually look like this
```
Comment (ahnv) [inline features/embed-video.md]:
```suggestion
Embedding a video from ImagKit allows you to integrate the video directly onto your platform without requiring an additional video player. Typically, the code to embed a video appears as:
```

**Review by imagekitio — CHANGES_REQUESTED** [review]
Please check https://developers.google.com/youtube/player_parameters
And rephrase.

File: features/embed-video.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+Embed video code can be generated for any video from the media library dashboard and from the files detail page.
+
+{% hint style="info" %}
+The embed feature is exclusive for published video files
+{% endhint %}
+
+#### Media library dashboard
+
+To generate the embed video code from the media library dashboard, hover over the lower right section of the video or right-click on the video file and select the "Embed" option.\
```
Comment (imagekitio) [inline features/embed-video.md]: Why is there an extra `\` at the end?

**Review by imagekitio — CHANGES_REQUESTED** [review]

File: features/embed-video.md (hunk 13 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got added while editing through gitbook , I will remove it

Comment (ankur-dwivedi) [reply features/embed-video.md]: done

File: features/embed-video.md (hunk 1 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got addressed in [c4e1ea7](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/c4e1ea7fa9f9f449b005a3430b5fdd26ce086949)

File: features/embed-video.md (hunk 2 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 3 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 4 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 5 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 6 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 7 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 8 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 9 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 11 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 12 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: got address in [2fc2641](https://github.com/imagekitio/docs-keep-a-changelog/pull/200/commits/2fc2641830c83c76f77ab7f4c11f39527c1aeabd)

File: features/embed-video.md (hunk 10 above)
Comment (ankur-dwivedi) [reply features/embed-video.md]: done

**Pull request merged**